src/qdrant_client/builers/vectors_config.rs

```diff
@@ -0,0 +1,51 @@
+use crate::qdrant::vectors_config::Config;
+use crate::qdrant::{VectorParams, VectorParamsMap, VectorsConfig};
+use std::collections::HashMap;
+
+const DEFAULT_VECTOR_NAME: &str = "";
+
+#[derive(Debug, Clone, Default)]
+pub struct VectorsConfigBuilder {
+    params: HashMap<String, VectorParams>,
+}
+
+impl VectorsConfigBuilder {
+    pub fn add_named_vector_params(
+        &mut self,
+        name: impl Into<String>,
+        params: impl Into<VectorParams>,
+    ) -> &mut Self {
+        self.params.insert(name.into(), params.into());
+        self
+    }
+
+    pub fn add_vector_params(&mut self, params: impl Into<VectorParams>) -> &mut Self {
+        self.params
+            .insert(DEFAULT_VECTOR_NAME.to_string(), params.into());
+        self
+    }
+}
+
+impl From<VectorsConfigBuilder> for VectorsConfig {
+    fn from(builder: VectorsConfigBuilder) -> Self {
+        if builder.params.is_empty() {
+            VectorsConfig { config: None }
+        } else if builder.params.len() == 1 {
+            if builder.params.contains_key(DEFAULT_VECTOR_NAME) {
```

Comment:
Let's combine these two conditions so that we can merge the two else-branches. They are the same.